techs/bemhtml.js

```diff
@@ -53,6 +53,8 @@ module.exports = require('./bem-xjst').buildFlow()
     .defineOption('naming')
     .defineOption('engineOptions')
     .defineOption('requires', {})
+    .defineOption('iife', false)
```

Comment:
I've got two questions about this option:
1. Maybe we can provide more generic solution like `wrap: { before: 'str', after: 'str' }` and even take a custom function to wrap initial content (see https://github.com/tadatuta/enb-wrap for reference)?
2. shouldn't it be introduced with separate PR or at least separate commit?

Reply:
@tadatuta 

1. why we might need more generic solution here?
2. OK, let’s split into two commits.

Reply:
we really need iife & source maps fast :) can't live without them